tensorflow/lite/micro/kernels/quantize_common.cc

```diff
@@ -25,6 +25,55 @@ limitations under the License.
 
 namespace tflite {
 
+TfLiteStatus PrepareQuantizeReference(TfLiteContext* context,
```

Comment:
Let's change micro/kernels/quantize.cc to use this function in the current PR as well.

Reply:
Yes, missed a commit I think. Should be ok now.